src/Cli/ConfigGenerator.cs

```diff
@@ -12,6 +12,7 @@
 using Azure.DataApiBuilder.Service;
 using Cli.Commands;
 using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
why do you need this? Did i miss code which needs this?

Reply:
it's for using methods from CollectionUtils.
![alt text](image.png)